recipes/conda-which/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "conda-which" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/kelvinou01/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  path: ../
+  sha256: d5558cd419c8d46bdc958064cb97f963d1ea793866414c025906ec15033512ed
```

Comment:
You have a sha256 mismatch. See:
https://conda-forge.org/docs/maintainer/adding_pkgs/#populating-the-hash-field